webapp/src/main/java/com/box/l10n/mojito/converter/PeriodConverter.java

```diff
@@ -12,11 +12,11 @@
  */
 @Component
 @ConfigurationPropertiesBinding
-public class PeriodConverter implements Converter<String, Period> {
+public class PeriodConverter implements Converter<String, Duration> {
 
   @Override
-  public Period convert(String source) {
-    long sourceAsLong = Long.valueOf(source);
-    return new Period(sourceAsLong);
+  public Duration convert(String source) {
+    long sourceAsLong = Long.parseLong(source);
```

Comment:
why that change? it is just because it is a primitive type, or are there other difference?

Reply:
yes, Long.valueOf returns Long and Long.parseLong returns the primitive type. It was a hint from the IDE that I just accepted.